src/components/Dashboard/TodoList.vue

```diff
@@ -189,7 +189,7 @@
 
                 <!-- draft correction or conversion or restoration or consent to continuation Out (staff only) -->
                 <template v-else-if="isStatusDraft(item) && (isTypeCorrection(item) || isTypeConversion(item) ||
-                  isTypeRestoration(item) || isTypeConsentContinuationOut(item)) && isRoleStaff"
+                  EnumUtilities.isTypeRestoration(item) || isTypeConsentContinuationOut(item)) && isRoleStaff"
```

Comment:
I only changed some of these "enum methods" over from EnumMixin to EnumUtilities. But I did delete the old ones along the way.